rust/src/lib.rs

```diff
@@ -450,6 +451,14 @@ pub mod ffi {
         fn get_packages(&self) -> &[String];
     }
 
+    extern "Rust" {
+        type RepoMetadataTarget;
+
+        fn is_inline(&self) -> bool;
```

Comment:
This is OK as is, but I think it would be more elegant to add [shared enum](https://cxx.rs/shared.html) here, and then `impl From<RepoMetadataTarget> for crate::ffi::RepoMetadataTarget { }` that converts them.

I've seen some RFEs go by for cxx.rs to make things like this easier.

Reply:
Done!